.github/workflows/ci.yml

```diff
@@ -47,28 +43,22 @@ jobs:
       image: 'erlang:${{ matrix.otp_vsn }}'
```

Comment:
I wonder if we can try to use `erlang:${{ matrix.otp_vsn }}-alpine`? But I think Erlang only supports alpine starting from OTP-20

Reply:
Let me check that for you.

Reply:
Yup, 19.x not available as `alpine`. Everything else available as `alpine`.

Reply:
I vote down this suggestion. I'm also curious to hear your pov @seriyps , but going ubuntu to debian already adds unnecessary friction that has little to do with Erlang (see minimum git version for github actions), and going debian to alpine could potentially add even more - if not now, later.

My pov is that obviously there might be reasons to deploy systems running on specific distros. Similarly, if you wanted to test software in different distros, you'd go for specific distros. But in this case, we're testing some Erlang sofware that we think has no distro-specific behaviour, so ideally I would test with the easiest distro to work with - Ubuntu.

@paulo-ferraz-oliveira you mentioned the setup-erlang action https://github.com/marketplace/actions/setup-erlang . A cursory read makes me assume that they download a "compiled" version (fast install times). Is that right? Did you try it out?

Reply:
I've already [tried it out](https://github.com/inaka/elvis_core/pull/143/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR20), but IIRC I stumbled upon a more [difficult version management](https://github.com/inaka/elvis_core/pull/143#issuecomment-744728919) (not all versions were available, for example). Want me to test it out here? I'm assuming you're looking for faster builds.

Reply:
My reasoning is more:
* there has to be one canonical way for erlang repos to test in CI, specifically github actions.
* that canonical way should probably be based on Ubuntu, for reasons of keeping the bar low

Speed should rarely be a goal, but it can be a show stopper - like when I tried to use kerl, and it would build OTP for 10+ minutes.

Let's merge this PR as is, because it is an improvement over what we have. Thank you for that.

IFF you want, file another PR with setup-erlang. From the looks of it, it could be that canonical solution that I'm referring to, and if you have already implemented it for inaka's repos, all the better.